packet/client.go

```diff
@@ -19,12 +19,16 @@ import (
 type hardwareGetter interface {
 }
 
+type workflowGetter interface {
+}
```

Comment:
this should not be an empty interface

Reply:
any suggestion, what we can place in here ?

Reply:
I understand why you need it but it is not a good practice. An empty interface means EVERYTHING. And that's too much. If we do not have a mock for the `workflowClient` we should probably create one. Or either way, the `workflowGetter` and the `hardwareGetter` should require the functions used by the underline components.

Reply:
@gianarb yes, make sense. I have added get set methods in interface. Currently this is only for workflow client. We can also use this to mock the client. If this works fine maybe we can do this for hardware client as well. 

commit : https://github.com/tinkerbell/boots/pull/50/commits/375a7166ea03e2a0266a822ac6078f1a9883fe80

Reply:
Yep, in my opinion, this works way better! Thanks